packages/framework/src/Component/Error/ErrorPagesFacade.php

```diff
@@ -97,6 +99,8 @@ public function getErrorPageStatusCodeByStatusCode($statusCode)
     {
         if ($statusCode === Response::HTTP_NOT_FOUND || $statusCode === Response::HTTP_FORBIDDEN) {
             return static::PAGE_STATUS_CODE_404;
+        } elseif ($statusCode === Response::HTTP_GONE) {
+            return static::PAGE_STATUS_CODE_410;
```

Comment:
_nitpick 🤓_

`switch` could be more readable here, but I'm ok with the current state too.

Reply:
39ed913e58030a684cd8a2f3ac8c194e934db3ec